### PR TITLE
Add profile pic and random image on button click

### PR DESCRIPTION
**Pull request by Angel-AG**
A little bit of extra content to the about me section.

### CODE REVIEW

**Review by mschillaci — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,16 +13,59 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * An array of dog pics' source and alternative text
+ * @const {Array.<{source: string, alt: string}>}
```
Comment (akihiroota87) [inline portfolio/src/main/webapp/script.js]:
```suggestion
 * @const {!Array<{source: string, alt: string}>}
```
I don't think you need a "."

**Review by akihiroota87 — APPROVED** [review]
Looks good to me!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (Angel-AG) [reply portfolio/src/main/webapp/script.js]: Understood, thanks!

**Pull request merged**